.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: 🐛 Bug Report
+description: Do you find a bug? Create a report to help us improve.
+title: "[BUG]:"
+labels: ["bug", "blocked"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a bug! Please fill out the details below to help us understand and resolve it.
+  
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Provide a clear and concise description of the bug. Include steps to reproduce the behavior, the expected result, and the actual result.
+      placeholder: "e.g., Clicking the submit button goes to 404. Steps to reproduce: 1. Click on X. 2. Observe Y. 3. Z happens instead of A."
```

```suggestion
      placeholder: "e.g., Clicking the link on this page ends up in a 404. I think that the link needs to be updated to point to <insert new link here>"
```

Perhaps a more documentation-focused example would help?